src/qz/printer/action/PrintHTML.java

```diff
@@ -43,16 +43,29 @@ public void parseData(JSONArray printData, PrintOptions options) throws JSONExce
 
                 PrintingUtilities.Format format = PrintingUtilities.Format.valueOf(data.optString("format", "FILE").toUpperCase());
 
-                double pageWidth = PrinterJob.getPrinterJob().getPageFormat(null).getWidth();
+                double pageZoom = pxlOpts.getDensity() / 72.0;
+                if (pageZoom <= 0) { pageZoom = 1; }
+
+                double pageWidth = 0;
+                double pageHeight = 0;
+
+                // web dimension use 96dpi (or equivalent dp/metric)
```

Comment:
The [MDN on length](https://developer.mozilla.org/en-US/docs/Web/CSS/length#Absolute_length_units) goes over how css dimensions correspond to physical media.